### PR TITLE
[Meshmodel] Update WASMFilter component

### PR DESCRIPTION
**Pull request by MUzairS15**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by MUzairS15** [comment]
// @Abhishek-kumar09

**Review by leecalcote — CHANGES_REQUESTED** [review]
Use the component generator... ?

**Comment by leecalcote** [comment]
Is this PR necessary because the component generator currently doesn't support writing to this location on filesystem or that it writes to the wrong model?

File: server/meshmodel/components/core/WASMFilter.json [hunk 1]
```diff
@@ -4,28 +4,28 @@
...
         }
     },
-    "schema": "{\"$id\":\"https:\/\/meshery.layer5.io\/system\/models\/definition\/Workload\/WASMFilter\",\"$schema\":\"http:\/\/json-schema.org\/draft-07\/schema\",\"properties\":{\"wasm-filter\":{\"type\":\"string\"},\"config\":{\"type\":\"string\"}},\"required\":[\"wasm-filter\"],\"title\":\"WASM Filter\",\"type\":\"object\"}"
+    "schema": "{\"$id\":\"https://meshery.layer5.io/system/models/definition/Workload/WASMFilter\",\"$schema\":\"http://json-schema.org/draft-07/schema\",\"properties\":{\"wasmFilter\":{\"description\":\"Publicy accessible WASM filter URL.\",\"type\":\"string\"},\"config\":{\"description\":\"The YAML config of the filter being patched.\",\"type\":\"string\",\"format\":\"textarea\"}},\"required\":[\"wasmFilter\"],\"title\":\"WASM Filter\",\"type\":\"object\"}"
```
Comment (leecalcote) [inline server/meshmodel/components/core/WASMFilter.json:30]:
```suggestion
    "schema": "{\"$id\":\"https://meshery.layer5.io/system/models/definition/Workload/WASMFilter\",\"$schema\":\"http://json-schema.org/draft-07/schema\",\"properties\":{\"wasmFilter\":{\"description\":\"Location (URL) of WASM filter accessible by Istio/Kubernetes.\",\"type\":\"string\"},\"config\":{\"description\":\"Configuration of the `spec.configPatches.patch.value` of the Istio Envoy Filter resource for your specific filter.\",\"type\":\"string\",\"format\":\"textarea\"}},\"required\":[\"wasmFilter\"],\"title\":\"WASM Filter\",\"type\":\"object\"}"
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**